### DISCUSSION TITLE
解读新思科技2021年OSSRA报告

### DISCUSSION THREAD

**Discussion by wangleo61** [kaiyuanshe/ONES#69] [日常讨论]
软件吞噬世界，开源吞噬软件，解读2021年OSSRA报告
https://www.eet-china.com/news/11464.html

**Comment by will-ww** [comment]
> 软件吞噬世界，开源吞噬软件，解读2021年OSSRA报告 https://www.eet-china.com/news/11464.html

收藏！

**Comment by wangleo61** [comment]
谢谢王老师

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: Will ***@***.***>
发送时间: Wednesday, April 6, 2022 11:12 AM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] 解读新思科技2021年OSSRA报告 (Discussion #69)


软件吞噬世界，开源吞噬软件，解读2021年OSSRA报告 https://www.eet-china.com/news/11464.html

收藏！

―
